backend/matching-service/app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "matching-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "npx tsc",
+    "start": "node dist/index.js",
+    "dev": "nodemon index.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@socket.io/redis-adapter": "^8.2.1",
+    "@socket.io/redis-streams-adapter": "^0.1.0",
```

Comment:
btw why do we need the redis adapters?

Reply:
woops was just testing it out will remove